Softeq.XToolkit.WhiteLabel.Forms/Converters/MultiplyConverter.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Softeq.XToolkit.WhiteLabel.Forms.Converters
+{
+    /// <summary>
+    ///     Converts integer value multiplying it by parameter.
+    /// </summary>
+    public class MultiplyConverter : IValueConverter
```

Comment:
Could you please help with a better understanding of which case it can be used?

Reply:
Added comment, but I agree that its usage might be a bit specific and we might not want to add it. Please see the use case I described